Soomla/Assets/Plugins/Soomla/Profile/SoomlaProfile.cs

```diff
@@ -882,14 +893,16 @@ public class SoomlaProfile
 		}
 
 		internal static void TryFireProfileInitialized () {
-			if (AllProvidersInitialized()) {
+			if (AllProvidersInitialized() && nativeModulesInitialized) {
 				ProfileEvents.OnSoomlaProfileInitialized();
 			}
 		}
 
 		/** PROTECTED & PRIVATE FUNCTIONS **/
 
-		protected virtual void _initialize(string customParamsJson) { }
+		protected virtual void _initialize(string customParamsJson) {
+			nativeModulesInitialized = true;
```

Comment:
@borisspektor, I understand, this implementation of `_instance` will run only in editor, else - in `SoomlaProfileIOS` or `SoomlaProfileAndroid`.